python/setup.py

```diff
@@ -55,7 +55,7 @@
     package_data={"looker_sdk": ["py.typed", "looker_sdk/looker-sample.ini"]},
     packages=find_packages(),
     # restrict python to <=3.9.9 due to https://github.com/looker-open-source/sdk-codegen/issues/944
-    python_requires=">=3.6, <=3.9.9",
+    python_requires=">=3.6",
```

Comment:
up up and away

Reply:
I think we can also delete the comment now in a followup